src/pokete/classes/difficulty_ui.py

```
@@ -15,7 +15,7 @@ def __init__(self):
         # We'll build the text dynamically in __call__
         super().__init__(se.Text(""), name="Director's Insight", info=[CloseLabel()])
 
-    def _get_insight_text(self):
+    def _get_insight_text(self, ctx):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_get_insight_text`

(ANN202)